package.json

```diff
@@ -12,7 +12,7 @@
     "ipfs-api": "^17.1.3",
     "ipfs-geoip": "^2.3.0",
     "ipfs-logo": "github:ipfs/logo",
-    "ipfsd-ctl": "^0.25.1",
+    "ipfsd-ctl": "^0.26.0",
```

Comment:
Did it work flawlessly for you? I'm having a hard time here https://github.com/ipfs/js-ipfs-api/pull/638

Reply:
Haven't tested it yet. Probably later today, but can't now, unfortunately. 😞

Reply:
@diasdavid I'm not having issues (to test I deleted every IPFS binary from my computer, removed the ipfs repo and just run `station`).

Reply:
You can add files still?

Reply:
Yes, I'm not having any issues.